config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-5-upgrades.yaml

```diff
@@ -299,9 +299,62 @@ periodics:
       - name: KUBERNETES_VERSION_UPGRADE_FROM
         value: "stable-1.27"
       - name: KUBERNETES_VERSION_UPGRADE_TO
-        value: "ci/latest-1.28"
+        value: "stable-1.28"
       - name: ETCD_VERSION_UPGRADE_TO
-        value: "3.5.7-0"
+        value: "3.5.9-0"
+      - name: COREDNS_VERSION_UPGRADE_TO
+        value: "v1.10.1"
+      - name: GINKGO_FOCUS
+        value: "\\[K8s-Upgrade\\]"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          cpu: 7300m
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.5
+    testgrid-tab-name: capi-e2e-release-1-5-1-27-1-28
+    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: "4"
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-28-latest-release-1-5
```

Comment:
I think we should not add this job to 1.5 as 1.5 will never support Kubernetes 1.29. So we also shouldn't test it and invest time to keep it green

Reply:
Please check if this is documented clear enough in release tasks

Reply:
Updating the docs here:

- https://github.com/kubernetes-sigs/cluster-api/pull/9209